pages/shop/index.js

```diff
@@ -55,6 +55,13 @@ const Shop = (props) => {
           font: inherit;
```

Comment:
The default behavior for font is to inherit it from parent itself, so we can remove this line

Reply:
I think that is for the cart button only, I have not made any changes to that button